src/apps/mesoscale-explorer/ui/states.tsx

```diff
@@ -211,15 +211,15 @@ export async function loadPdb(ctx: PluginContext, id: string) {
     await createHierarchy(ctx, data.ref);
 }
 
-export async function loadPdbDev(ctx: PluginContext, id: string) {
+export async function loadPdbIhm(ctx: PluginContext, id: string) {
```

Comment:
How about leaving both functions for backwards compat for now and remove in 5.0?

Reply:
Sounds good, updated the code to keep both functions around while marking them as deprecated as suggested.